include/small/util.h

```diff
@@ -49,6 +49,20 @@
 #  define small_unlikely(x)  (x)
 #endif
 
+/**
+ * Add `always_inline` attribute to the function if possible. Add inline too.
+ * Such function will always be inlined.
+ *
+ * Usage example:
+ *
+ * static SMALL_ALWAYS_INLINE void f() {}
+ */
+#if __has_attribute(always_inline) || defined(__GNUC__)
```

Comment:
You have to define [`__has_attribute`](https://github.com/tarantool/tarantool/blob/abefe2c6a2c890c1ee1549c26060a1487b546f60/src/trivia/util.h#L186-L193) if it's absent.

Reply:
Done.